### ISSUE TITLE
Use setpriv instead

### ISSUE THREAD

**Issue by obadz** [RhysU/yeshup#1]
Not an issue per say but just to flag that one can use `setpriv --pdeathsig HUP [command]` to achieve the same result as this program.

https://man7.org/linux/man-pages/man1/setpriv.1.html

Cheers

**Comment by RhysU** [comment]
Thank you. That's helpful.

**Comment by obadz** [comment]
No: thank YOU. I used your program for years until I figured this out :-)

**Comment by RhysU** [comment]
Glad it was useful!

You might consider upstreaming this information to the Stack Overflow thread linked from the Readme.

**Comment by obadz** [comment]
Good idea. https://stackoverflow.com/a/69449340/2029548
